src/styles/blocks/card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  width: $main-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__imac-image {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__content {
+    font-family: Roboto, sans-serif;
```

Comment:
According to the checklist's code style rules, font-family should be defined in a variable for reusability and to ensure consistency across the project. Consider creating a variable for the font-family.